examples/web3.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback } from 'react';
+
+import {
+  useEthereumProvider,
+  EthereumOnboardingOptions,
+  PlaidWeb3OnSuccess,
+  EIP1193Provider,
+} from 'react-plaid-link/web3';
+
+import { PlaidLinkOnEvent, PlaidLinkOnExit } from 'react-plaid-link';
+
+const PlaidWeb3Link = () => {
+  const token = '';
+
+  const [walletProvider, setWalletProvider] = React.useState<EIP1193Provider>();
+  const [addresses, setAddresses] = React.useState<string[]>();
+  const [checkedExistingProvider, setCheckedExistingProvider] = React.useState(
+    false
+  );
+
+  const onSuccess = useCallback<PlaidWeb3OnSuccess>(
+    async provider => {
+      setWalletProvider(provider);
+    },
```

Comment:
would be nice to show use of metadata in the example code

Reply:
I left it out because we don't expose anything interesting in metadata right now (just link_session_id) and don't document it on the main docs yet either - but I did remove `onEvent` which we also don't emit anything to right now!